tests/vmcheck/test-override-replace-2.sh

```diff
@@ -159,10 +159,15 @@ case $versionid in
   # XXX: this isn't actually the gold selinux; that one is too old for
   # container-selinux and moby-engine. rather than trying to change multiple
   # packages, we use one that's in coreos-pool since that also prevents GC
+  41)
+    evr=38.28-1.fc39
+    koji_url="https://koji.fedoraproject.org/koji/buildinfo?buildID=2291271"
```

Comment:
This needs to be different for f41

Reply:
Ok updated using packages from: https://kojipkgs.fedoraproject.org/repos-dist/coreos-pool/6526564/x86_64/Packages/ 
Let's see if we finally see all green.